packages/w3up-client/src/space.js

```diff
@@ -1,47 +1,99 @@
 export * from '@web3-storage/access/space'
+import * as Usage from './capability/usage.js'
+import * as API from './types.js'
 
-export class Space {
-  /** @type {import('./types.js').DID} */
-  #did
+/**
+ * @typedef {object} Model
+ * @property {API.SpaceDID} id
+ * @property {{name?:string}} [meta]
+ * @property {API.Agent} agent
+ */
 
-  /** @type {Record<string, any>} */
-  #meta
+export class Space {
+  #model
 
   /**
-   * @param {import('./types.js').DID} did
-   * @param {Record<string, any>} meta
+   * @param {Model} model
    */
-  constructor(did, meta = {}) {
```

Comment:
changes API. Should we either:
* support the old constructor signature too
* make this a major version bump
* note the breaking change but dont do more than minor (as is)

Any is fine, just noting the API change here